### PR TITLE
Scripts update

### PR DESCRIPTION
**Pull request by brkomir**
- introduced common scripts for keeper actions - rebalance, reallocate, and exit all positions
- updated the deployment scripts to use create for deterministic deploys 
- added deploy and keeper scripts for scUSDT and scSDAI
- other refactoring and improvements to all script files

### CODE REVIEW

File: script/base/MainnetAddresses.sol
Comment (dxganta) [inline script/base/MainnetAddresses.sol:1]: You can add the other required deployed contracts too

/   scUSDT: 0x237eCDF745d2a0052AeaF6f027ce82F77431871E
//   scUSDT AaveV3Adapter: 0x795FDAC1bB26D219991f1923190cA7eB0046e32E
//   scUSDT swapper: 0xcD0Ae497068980ce94fbe20A6F7431329F0e4D08
//   scUSDT priceConverter: 0x8e045458385d51d047bCA5b2B3b74053e235f5F9

//   scSdai: 0x0Fc97657B67C7E7bD4100c72851d0377DA14B470
//   scDai: 0x16f3cdA06743a58bDdE123687F99E80DCbc28d14
//   scSdai sparkAdapter: 0xeD81a1D7cb2AdC2ee379B594e0dA6a988a865ae4
//   scSdai swapper: 0x3F76D80dfc54677262FeA7E94a73D96aC69F4A10
//   scSdai priceConverter: 0x0dD22Ebc502f328b29a19Bf0A74373B03DfD374c

File: script/v2/keeper-actions/RebalanceScUsdt.s.sol [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    uint256 public aaveV3InvestableAmountPercent = 1e18; // 100%
+    uint256 public aaveV3TargetLtv = 0.65e18; // 65%
+
+    /*//////////////////////////////////////////////////////////////*/
+
+    function _initializeAdapterSettings() internal override {
+        adapterSettings.push(
+            AdapterSettings({
+                adapterId: 1, // TODO: IAdapter(MainnetAddresses.SCUSDT_AAVEV3_ADAPTER).id(),
```
Comment (dxganta) [inline script/v2/keeper-actions/RebalanceScUsdt.s.sol:42]: What about this TODO?

**Comment by dxganta** [comment]
A lot of these contracts have been already deployed. Do we still have any use for the deploy scripts of those contracts?

**Comment by brkomir** [comment]
> A lot of these contracts have been already deployed already. Do we still have any use for the deploy scripts of those contracts?

I would say we do since we want to maintain consistency with our deployments utilizing CREATE2. At the very least, these scripts are using common deployment logic tested through these specific deployment scripts, so even if we end up not using them in the future, they are pinning down this base logic.

File: script/base/MainnetAddresses.sol
Comment (brkomir) [reply script/base/MainnetAddresses.sol:1]: [updated](4870e08a5b654975f270f4c125771f1f989c4777)

File: script/v2/keeper-actions/RebalanceScUsdt.s.sol (hunk 1 above)
Comment (brkomir) [reply script/v2/keeper-actions/RebalanceScUsdt.s.sol]: fixed

**Review by dxganta — APPROVED** [review]

**Pull request merged**